src/PrettyFormatter.go

```diff
@@ -56,16 +56,28 @@ func (prettyFormatter *PrettyFormatter) addKey(key string) {
 
 func (prettyFormatter *PrettyFormatter) createSetForPlanet(planet Planet) {
 	var completeTable = make(map[string]string)
+	id := planet.id
+	name := planet.name
+	planetType := planet.planetType
 	address := fmt.Sprintf("%s@%s", planet.user, planet.host)
 	number := strconv.Itoa(planet.outputStruct.position)
+	output := planet.outputStruct.output
+	if planet.outputStruct.errored || !planet.valid {
+		number = makeRed(number)
+		id = makeRed(id)
+		name = makeRed(name)
+		address = makeRed(address)
+		planetType = makeRed(planetType)
+		output = makeRed(planet.outputStruct.output)
+	}
 	prettyFormatter.addEntry("Nr.", number, completeTable)
-	prettyFormatter.addEntry("ID", planet.id, completeTable)
-	prettyFormatter.addEntry("Name", planet.name, completeTable)
+	prettyFormatter.addEntry("ID", id, completeTable)
+	prettyFormatter.addEntry("Name", name, completeTable)
 	prettyFormatter.addEntry("Address", address, completeTable)
-	prettyFormatter.addEntry("Type", planet.planetType, completeTable)
```

Comment:
how about renaming the makeRed() method to color(text string, flag bool). If flag is set then a color is applied otherwise the text is returned as is. 
adding new variables is not necessary IMHO.